README.md

```diff
@@ -63,6 +63,8 @@ $ nvs use chakracore-nightly
 
 ### Building Node-ChakraCore
 
+Node-ChakraCore can be build on Windows, Linux and Mac OS.
```

Comment:
This should still be "Node-ChakraCore can be ***built*** on Windows, Linux, and ***macOS***"

Reply:
fixed.